### PR TITLE
Adding Cyclic LR scheduling for Image Classification API 

### PR DESCRIPTION
**Pull request by harshithapv**
This PR implements Cyclic Learning rate scheduler. The following paper has more information.
https://arxiv.org/abs/1506.01186

Also added unit test and integration test for the same.

### CODE REVIEW

**Comment by codemzs** [comment]
>         public static void Main(string[] args) => RunAll();

please revert this file.

---
Refers to: docs/samples/Microsoft.ML.Samples.GPU/Program.cs:9 in 7fd1769. [](commit_id = 7fd17699f921aeea0eb0eb56c969076b6a6d691c, deletion_comment = False)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@
 using System.Threading.Tasks;
 using Microsoft.ML;
 using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs]: please revert this file and instead add a new sample if needed.

File: docs/samples/Microsoft.ML.Samples.GPU/Program.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ internal static void RunAll()
         {
             int samples = 0;
             var types = Assembly.GetExecutingAssembly().GetTypes();
-            foreach (var type in types)
```
Comment (bpstark) [inline docs/samples/Microsoft.ML.Samples.GPU/Program.cs]: This file should be reverted to run all samples again

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs (hunk 1 above)
Comment (harshithapv) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs]: Reverted.

File: docs/samples/Microsoft.ML.Samples.GPU/Program.cs (hunk 2 above)
Comment (harshithapv) [reply docs/samples/Microsoft.ML.Samples.GPU/Program.cs]: Reverted.

**Comment by michaelgsharp** [comment]
@harshithapv I think this looks good to me. However, since the time this PR was created we now longer have the netcoreapp 30 build, we have updated to 3.1 instead. It looks like github can't handle that well, so we need to close this PR and create a new one so it triggers the right builds. I'm going to go ahead and close this PR, can you create a new one so we can get this in?

**Pull request closed without merging**